client/me/purchases/concierge-banner/style.scss

```
@@ -1,51 +1,58 @@
 .concierge-banner__placeholders.is-placeholder {
-    display: flex;
-    flex-direction: column;
+	display: flex;
+	flex-direction: column;
 
-    @include breakpoint-deprecated( '>960px' ) {
-        flex-direction: row;
-    }
+	@include breakpoint-deprecated( '>960px' ) {
+		flex-direction: row;
+	}
 }
 
 .concierge-banner__placeholder-row {
-    flex: 0 0 100%;
-    height: 20px;
-    margin-bottom: 10px;
+	flex: 0 0 100%;
+	height: 20px;
+	margin-bottom: 10px;
 }
 
 .concierge-banner__placeholder-row-container {
-    flex: 0.8;
+	flex: 0.8;
 
-    .is-placeholder {
-        animation: pulse-light 800ms ease-in-out infinite;
-        background: var( --color-neutral-10 );
-    }
+	.is-placeholder {
+		animation: pulse-light 800ms ease-in-out infinite;
+		background: var( --color-neutral-10 );
+	}
 }
 
 .concierge-banner__placeholder-button-container {
-    flex: 0.2;
+	flex: 0.2;
 
-    .is-placeholder {
-        animation: pulse-light 800ms ease-in-out infinite;
-        background: var( --color-neutral-10 );
-    }
+	.is-placeholder {
+		animation: pulse-light 800ms ease-in-out infinite;
+		background: var( --color-neutral-10 );
+	}
 
-    @include breakpoint-deprecated( '>960px' ) {
-        margin: auto;
-    }
+	@include breakpoint-deprecated( '>960px' ) {
+		margin: auto;
+	}
 }
 
 .concierge-banner__placeholder-button {
-    height: 30px;
-    width: 80px;
-
-    @include breakpoint-deprecated( '>960px' ) {
-        height: 30px;
-        margin: auto;
-        width: 80px;
-    }
-
-    @include breakpoint-deprecated( '<480px' ) {
-        width: 100%;
-    }
+	height: 30px;
+	width: 80px;
+
+	@include breakpoint-deprecated( '>960px' ) {
+		height: 30px;
+		margin: auto;
+		width: 80px;
+	}
+
+	@include breakpoint-deprecated( '<480px' ) {
+		width: 100%;
+	}
+}
+
+.banner__action {
```

Please wrap this selector, otherwise it can lead to hard-to-diagnose Heisenbugs that depend on the load sequence for different components. We'll need to use the `className` prop on the `Banner` component to support that.
```suggestion
.concierge-banner .banner__action {
```

---

I am also not sure why we need this rule. Could you share the reason for it? My understanding is that we should let long button text wrap and result in multi-line buttons when that is necessary.

---

Also, if we do need to keep this rule, then I think we need to apply it only for larger screen sizes, as we should let the text overflow on narrower screens/mobile, where we default to having the CTA span the entire banner below the main banner text.

---

We discussed this in Slack. To avoid button text wrapping, we will use `min-width: 120px` as this text could be quite long in some languages and `whitespace: nowrap` might not always be appropriate.